services/src/app.js

```diff
@@ -128,4 +128,9 @@ app.configure(jwt({
   }
 }));
 
+setInterval(() => {
+  logger.info('Sending ping');
+  app.service('status').emit('ping', { 'timestamp' : (new Date()) });
```

Comment:
AIUI `Date.now()` seems now preferrable to `new Date()`, since ES 5.1. As it does not need creating a new object.